### DISCUSSION TITLE
Recommendation software for accesible challenge

### DISCUSSION THREAD

**Discussion by vala220805** [msusdev/Innovation-Challenge#6] [Q&A]
Hi,

Our team has a question about the "hack for accessible personalization and recommendation". Should our project focus on addressing multiple different abilities, or is it acceptable to focus on just one specific ability?

Additionally, we're a bit confused about the challenge's requirements. Does our project need to be specifically a recommendation system, or can it be any type of software designed to empower people with disabilities?

Thank you!

**Comment by MatthewCalder-msft** [comment]
Focusing on one specific ability would be totally fine. Probably best to start there and then try to address more if you have time. (Even just having a plan to address more would be seen favorably by the judges). 
Your project does need to make a recommendation experience more accessible. I'd recommend taking an available open source project and adding features to it. [Here are some to maybe start evaluating ](https://github.com/topics/movie-recommendation-system)
Also, tomorrow we have a talk with an accessibility expert from Microsoft who may be helpful. [Building for Accessibility with Azure](https://www.youtube.com/watch?v=FwvUmlAAIz0)

**Reply by MatthewCalder-msft** [reply]
I also wanted to share this link to the [Microsoft News Recommendation data set](https://learn.microsoft.com/en-us/azure/open-datasets/dataset-microsoft-news?tabs=azureml-opendatasets). Could be helpful!
